etc/ci-coveralls-merge.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+# Hacked up merger for testing
+
+import json
+import os
+import os.path
+
+print("Merging coveralls json coverage") 
+merged = { 'service_name': 'travis-ci'
+         , 'service_job_id': os.environ['TRAVIS_JOB_ID']
+         , 'parallel': 'true'
+         , 'source_files': [] }
+
+for fn in ['gap-coveralls.json', 'c-coveralls.json']:
+    print "file: %s" % (fn,),
+    if os.path.isfile(fn):
+        f = open(fn, 'r')
```

Comment:
It would be more "pythonesque" to use a `with` statement, here, i.e.,
```python
  with open(fn, 'r') as f:
    cover = json.load(f)
    merged['source_files'].extend(cover['source_files'])
```
And then there is no need for `f.close()`